scripts/artist_ids.coffee

```diff
@@ -0,0 +1,612 @@
+module.exports = 
+
+ARTISTIDS = [
```

Comment:
I think we can prolly just re-use the one file between this and https://github.com/artsy/positron/pull/668/files#diff-9910aafd47cba1f0a8336ed3b16f537cR3